tests/testthat/test-get-country-regional-data.R

```diff
@@ -0,0 +1,53 @@
+# Tests each dataset returned by countries specified in get_info_covidregionaldata
+source("custom_tests/get_data.R")
+
+# 1. Test data.frame is returned
+test_that("Level 1 data returned in dataframes", {
+  skip_on_cran()
+  
+  level1_dataframes <- purrr::map(data_level_1, ~ class(.x)) %>%
+    purrr::keep(~ any(grepl("data.frame", .x))) %>%
+    length()
+  
+  expect_equal(level1_dataframes, length(countries_level_1))
+})
+
+# 2. Test essential column types are appropriate
+test_that("Level 1 data has correct col types for date, region, cases", {
+  skip_on_cran()
+  
+  for (country in countries_level_1) {
```

Comment:
perhaps make the test a function and loop over the countries inserting the country name into the test so that we can easily identify where is causing the issue?